build.sh

```diff
@@ -146,15 +146,27 @@ generate_event_logging_sources()
 # Event Logging Infrastructure
    __GeneratedIntermediate="$__IntermediatesDir/Generated"
    __GeneratedIntermediateEventProvider="$__GeneratedIntermediate/eventprovider_new"
+   __GeneratedIntermediateEventPipe="$__GeneratedIntermediate/eventpipe_new"
```

Comment:
Yes, it seems this error only happens on Windows platforms. I guess the reason is that in file eventpipeprovider.h line 16, the enumerate variable `Error ` may be defined somewhere else. If we use `enum class EventPipeEventLevel` instead of `enum EventPipeEventLevel`, this error will disappear. Of course, we need to use` EventPipeEventLevel::Critical` instead of` Critical` if we make this change. Make the change?